test/oscar/compiler/helloWorld.oscar.out

```diff
@@ -0,0 +1 @@
+Hello World
```

Comment:
does `println` print a newline after the string? is this something to check here?